### DISCUSSION TITLE
Turn On/Off 2007-2012 Theme!

### DISCUSSION THREAD

**Discussion by AdyOsu** [ppy/osu#26623] [Ideas]
i'd really love to see this feature in lazer an option like "2007-2012 theme or look" that brings back the old animations, the old osu! logo, combo fire, and everything that was around back then! it would really make the old maps much fun, and nicer, especially the skins that were made them and i actually still prefer the old look then the modern one! 
this is from osu feature request btw
https://osu.ppy.sh/community/forums/topics/1825157?n=1

**Comment by home-gihub** [comment]
Lazer has a 2008 gameplay skin but no combo fire/burst and the main menu isnt skinnable at all

**Comment by voiceofthunder0108** [comment]
peppy isn’t interested in that, he doesn’t care about the old players who built this game and supported it for years. He prefers to target the new generation of players from tiktok and twitter, who will quit the game after a few months.

Some time ago, I made a [similar request](https://github.com/ppy/osu/discussions/33533) and got the reply: 

> “Thanks for the feedback! Stable will still remain for those that prefer the old look. Meanwhile, skins will improve.”

 So there’s no point in getting your hopes up. If you want to play on the old clients, you can do it offline... or elsewhere...

**Reply by ziv-vy** [reply]
> Meanwhile, skins will improve

This means making lazer fully skinnable. It will certainly happen. The devs are aiming to make lazer way more skinnable than stable ever dreamt to be. Just not as high priority as other stuff like matchmaking or making stock designs good.

**Reply by voiceofthunder0108** [reply]
> This means making lazer fully skinnable. It will certainly happen. The devs are aiming to make lazer way more skinnable than stable ever dreamt to be. Just not as high priority as other stuff like matchmaking or making stock designs good.

Idk, maybe. I’d rather not expect anything at this point and even if it actually does happen, it’ll probably take many years. I’ve been playing osu since 2013, so I’m well aware of how long it takes for any changes to happen. Until then, I’ll just play “offline,” because I value the old vibe, and that’s something I can only get with the old clients.

Peppy’s response is unclear to me and I interpret it as, “we’re not planning to do anything about it, wait 10 years until lazer can be fully skinned,” but I doubt that even then it’ll be possible to make lazer look anything like the old clients did.

**Comment by kyiuu1** [comment]
ppy dont like everything can be customized by player, so impossible. maybe consider turn to a third-party ruleset

**Reply by ziv-vy** [reply]
> ppy dont like everything can be customized by player, so impossible

Why would he not like that?
